amenity-points.mss

```diff
@@ -2821,7 +2828,6 @@
     }
   }
 
-  [feature = 'amenity_nursing_home'],
```

Comment:
It looks like you've accidentally deleted this line containing`[feature = 'amenity_nursing_home'],`

Reply:
Thanks for these, they're now fixed